website/docs/r/cosmosdb_account.html.markdown

```diff
@@ -219,8 +219,7 @@ A `cors_rule` block supports the following:
 
 A `identity` block supports the following:
 
-* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this Cosmos Account. The only possible value is `SystemAssigned`.
-
+* `type` - (Required) The Type of Managed Identity assigned to this Cosmos account. Possible values are `SystemAssigned`, `UserAssigned` and `SystemAssigned, UserAssigned`.
```

Comment:
We'll want an empty line below this, else this'll become an `h1` fwiw:

```suggestion
* `type` - (Required) The Type of Managed Identity assigned to this Cosmos account. Possible values are `SystemAssigned`, `UserAssigned` and `SystemAssigned, UserAssigned`.

```

Also since we're adding the other fields here by changing the Identity type, can we output those here too?